### PR TITLE
fixed search items in meeting rooms

### PR DESCRIPTION
**Pull request by Goritam**

### CODE REVIEW

File: ESO/ESO/Modules/Floor/BaseTabBarController.swift [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    }
+    
+    func didMoveToPage(_ controller: UIViewController, index: Int) {
+        setUserContext(index: index)
+        listItem.moveToPage(index)
+        mapItem.moveToPage(index)
+    }
+    
+    func willMoveToPage(_ controller: UIViewController, index: Int) {
+
```
Comment (MihailBohnearschi) [inline ESO/ESO/Modules/Floor/BaseTabBarController.swift]: Is this method supposed to be empty?

File: ESO/ESO/Modules/Floor/BaseTabBarController.swift [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        searchBar.searchTextDidChanged = {  searchString in
+            var placesForSearch = self.placesByFeature.flatMap({$0})
+            if !searchString.isEmpty{
+                placesForSearch =  placesForSearch.filter({ (place) -> Bool in
+                    place.searchString.contains(find: searchString)
+                })
+                self.searchViewController.update(places: placesForSearch)
+            }
+            else {
+                self.searchViewController.update(places: [Place]())
```
Comment (gugell) [inline ESO/ESO/Modules/Floor/BaseTabBarController.swift]: [] is the best option here

File: ESO/ESO/Modules/Floor/PlacesList/PlacesListViewController.swift [hunk 3]
```diff
@@ -25,7 +25,6 @@ class PlacesListViewController: UIViewController, FloorItemDelegate {
         self.presenter = PlacesListPresenter(view: self, places: self.places)
         delegate = FloorPlacesTableDelegate(self)
         datasource = FloorDatasource(items: (presenter?.places)!, tableView: tableView, delegate: delegate)
```
Comment (gugell) [inline ESO/ESO/Modules/Floor/PlacesList/PlacesListViewController.swift]: lets unwrap all the optionals in the project

Comment (gugell) [inline ESO/ESO/Modules/Floor/PlacesList/PlacesListViewController.swift]: presenter should be as PlacesListPresenter! , not optional.

File: ESO/ESO/Modules/Floor/BaseTabBarController.swift (hunk 1 above)
Comment (gugell) [inline ESO/ESO/Modules/Floor/BaseTabBarController.swift]: that method is part of delegate protocol , it can be empty for now.

**Pull request merged**